relay-server/src/utils/unreal.rs

```diff
@@ -284,7 +284,7 @@ pub fn process_unreal_envelope(
     }
 
     // If we have UE4 info, ensure an event is there to fill. DO NOT fill if there is no unreal
-    // information, or otherwise `EnvelopeProcessor::process` breaks.
+    // information, or otherwise `EnvelopeProcessorService::process` breaks.
```

Comment:
nit: turn this into a proper link so that building docs will break if things change:

```suggestion
    // information, or otherwise [`EnvelopeProcessorService::process`] breaks.
```

Reply:
This is not a doc comment but a regular code comment, so it won't be checked.

Reply:
doh, yes